robo-components/DeploymentTrait.php

```diff
@@ -254,8 +251,13 @@ public function deployPantheon(string $branch_name = 'master', ?string $commit_m
     $this->_exec("cp web/sites/default/settings.pantheon.php $pantheon_directory/web/sites/default/settings.php");
 
     // Prevent attackers to reach these standalone scripts.
-    $this->_exec("rm -rf web/core/install.php");
-    $this->_exec("rm -rf web/core/update.php");
+    $this->_exec("rm -rf $pantheon_directory/web/core/install.php");
+    $this->_exec("rm -rf $pantheon_directory/web/core/update.php");
```

Comment:
```suggestion
    $this->_exec("rm -f $pantheon_directory/web/core/install.php");
    $this->_exec("rm -f $pantheon_directory/web/core/update.php");
```

We dont need `recursive delete` flag on individual files.